templates/bet/Inspector.tsx

```
@@ -0,0 +1,225 @@
+
+    const { privacy, claim, opponent, arbitrator, asset, amount, owner } = config
+
+    const fid = useFarcasterId()
+    const username = useFarcasterName()
+    const uploadImage = useUploadImage()
+
+    const [opponentInput, setOpponentInput] = useState('')
+    const [arbitratorInput, setArbitratorInput] = useState('')
+
+    // Set owner if it's not already set
+    useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: updateConfig

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)